.github/workflows/npm.yml

```diff
@@ -30,49 +34,35 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
+      
+      - name: Install dependencies
+        run: npm ci
 
       - name: Resolve latest contracts
-        working-directory: ./solidity
-        run: npm update @keep-network/keep-core @keep-network/sortition-pools
-
-      - name: Bump up version
         working-directory: ./solidity
         run: |
-          name=$(jq --raw-output .name package.json)
-          version=$(jq --raw-output .version package.json)
-          preid=$(echo $version | sed -e s/^.*-\\\([^.]*\\\).*$/\\1/)
-
-          # Check resolved `preid`. Currently only `pre` value is supported,
-          # other types of releases are not handled by this job.
-          if [ "$preid" != pre ]; then
-            echo "Unsupported preid. Resolved info:"
-            echo "$name@$version ; preid $preid"
-            exit 1
-          fi
+          npm update \
+            @keep-network/keep-core \
```

Comment:
We should also update `package.json` file to reflect the `dev` versions ranges.

Reply:
Right, this is going to be delivered in separate PR: https://github.com/keep-network/keep-ecdsa/pull/797.